app/js/lib/foxbox/services/ip-camera.js

```diff
@@ -14,10 +14,6 @@ export default class IpCameraService extends BaseService {
     return TYPE;
   }
 
-  getImageList() {
-    return this.get('image_list');
-  }
```

Comment:
`turnOn` and `turnOff` were syntactic sugar on top of the setters. I think we need to add methods for all the getters and setters of the adapter, so that we don't have to debug the box responses.
That'll be valuable too when we generate documentation from the code.